strawberryfields/configuration.py

```diff
@@ -143,6 +143,8 @@ def get_config_filepath(filename="config.toml"):
         if os.path.exists(filepath):
             return filepath
 
+    return None
```

Comment:
This was readded to satisfy the linter :smile:

Reply:
:thinking: